build/insecureinc/Dockerfile

```diff
@@ -1,8 +1,3 @@
-FROM tomcat:8.5-jre8
-RUN rm -rf /usr/local/tomcat/webapps/ROOT
+FROM vancleff/tomcat:8_5_jre_8
```

Comment:
@kantsk11 could we use an official image instead such as https://hub.docker.com/_/tomcat instead and install gcc on top?

` RUN  rm -rf /var/lib/apt/lists/* && \
    apt-get update --fix-missing && \
    apt-get -y install gcc`

Reply:
Thanks Paul, I've updated the InsecureInc image.